src/WebJobs.Extensions.DurableTask/Listener/TaskEntityShim.cs

```diff
@@ -166,7 +171,9 @@ public override async Task<string> Execute(OrchestrationContext innerContext, st
                 this.Config.GetIntputOutputTrace(serializedInput),
                 FunctionType.Entity,
                 isReplay: false);
-
+#if !FUNCTIONS_V1
```

Comment:
I think we can get rid of this chunk, because it will always be overwritten as `Completed` or `Failed`.

Reply:
I think you're right. Although I wonder if it would make sense for an Entity to ever be just "Running". What if its Locked?

Reply:
I removed it for the time being, but I think the discussion is worthwhile

Reply:
On that note, I'm now also unsure if my current approach event captures a "Running" Orchestrator

Reply:
In addition, I'm unsure how to test this for failure states. It would be good if we had that discussion.

Reply:
This appears to report a different status to AppInsights than what is the DTFx status of the orchestration (as visible in the status table)? 

Entities are "Running" while they wait for messages to arrive, which is most of the time. They should never be "Completed" or "Failed" which would indicate a serious bug. 

If we want to provide a different information, we should perhaps not use the same enumeration because I think that may be confusing?

Reply:
Thanks for pointing this out! Indeed, that would be confusing.
For cross-referencing, what and where is the status table you're referencing?

Reply:
Did we ever make a decision on this?

Reply:
Not yet. We should discuss it early next week.

Reply:
Ended up dropping updating the status for Entities, so they're always "running". I could also just completely avoid showing a status for entities, we should discuss.

Reply:
@sebastianburckhardt , does that sound appropriate to you?